src/NewRelicExceptionHandler.php

```diff
@@ -77,4 +77,13 @@ protected function logException(Exception $e)
         }
     }
 
+    /**
+     * Original handler's function
+     *
+     * @param Exception $e
+     */
+    public function shouldReport(Exception $e)
+    {
+        return $this->primaryHandler->shouldReport($e);
```

Comment:
There is no property named primaryHandler so not sure how this is supposed to work?

Reply:
🤔 This is really a problem, although in README the default primaryHandler is `App\Exceptions\Handler`.

Reply:
You can create a PR against https://github.com/digiaonline/lumen-chained-exception-handler, that's what's used here